services/fdb/server/fdbserver.go

```diff
@@ -0,0 +1,92 @@
+/* Copyright (c) 2023 Snowflake Inc. All rights reserved.
+
+   Licensed under the Apache License, Version 2.0 (the
+   "License"); you may not use this file except in compliance
+   with the License.  You may obtain a copy of the License at
+
+     http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing,
+   software distributed under the License is distributed on an
+   "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+   KIND, either express or implied.  See the License for the
+   specific language governing permissions and limitations
+   under the License.
+*/
+
+package server
+
+import (
+	"context"
+
+	"github.com/Snowflake-Labs/sansshell/services"
+	pb "github.com/Snowflake-Labs/sansshell/services/fdb"
+	"github.com/Snowflake-Labs/sansshell/services/util"
+	"github.com/Snowflake-Labs/sansshell/telemetry/metrics"
+	"go.opentelemetry.io/otel/attribute"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+)
+
+var (
+	// FDBServer string = "/usr/sbin/fdbserver" // uncomment that if fdbserver binary is in that location
+	FDBServer string
```

Comment:
Wait, how does it work? I don't see anywhere where you set it?

Note - if it's just defined in a code it should probably be a constant.

Reply:
When I tested the code, just uncommented the line 33 since it defines the fdbserver binary. The reason I uncomment is binary path may be different and to keep consistent with fdbcli: https://github.com/Snowflake-Labs/sansshell/blob/ec9a0e4af5597f44daba1df5ec688b43aecdfd27/services/fdb/server/fdbcli.go#L44 which is set somewhere else.

Reply:
Right, and FDBCLI is set via flag which is commented out here: https://github.com/Snowflake-Labs/sansshell/blob/c4a4183c060a1f9323278c07ae478509a4d79f43/cmd/sansshell-server/main.go#L96

I'm not sure why all FDB stuff is commented out by default, makes it kind annoying. Let't check with people why may know.

Reply:
I will submit a separate PR for uncommenting these lines just in case we may need to revert the PR.